src/data-mgt/node/controllers/transform.js

```diff
@@ -148,7 +148,7 @@ const data = {
                   cacheID,
                   JSON.stringify({ isCache: true, ...responseJSON })
                 );
-                redis.expire(cacheID, 86400);
+                redis.expire(cacheID, 3600);
```

Comment:
Also here! 3600 or 300 or 30??. just being curious!

Reply:
these are seconds. So 3600 refers to one hour. 300 is 5 minutes, 30 is 30 seconds. :)

Reply:
these are seconds. So 3600 refers to one hour. 300 is 5 minutes, 30 is 30 seconds. :)

Reply:
ah okay... so those values are the correct ones! got it thanks @Baalmart